src/System.Reflection.Metadata/tests/PortableExecutable/DebugDirectoryTests.cs

```diff
@@ -153,10 +153,12 @@ private void ValidateEmbeddedPortablePdb(PEReader reader)
             Assert.Equal(DebugDirectoryEntryType.Reproducible, entries[1].Type);
             Assert.Equal(DebugDirectoryEntryType.EmbeddedPortablePdb, entries[2].Type);
 
-            var provider = reader.ReadEmbeddedPortablePdbDebugDirectoryData(entries[2]);
-            var pdbReader = provider.GetMetadataReader();
-            var document = pdbReader.GetDocument(pdbReader.Documents.First());
-            Assert.Equal(@"C:\Documents.cs", pdbReader.GetString(document.Name));
+            using (var provider = reader.ReadEmbeddedPortablePdbDebugDirectoryData(entries[2]))
```

Comment:
Nit: can you use the actual type rather than var?  I had to go search for this method to figure out what the type of provider was